app/models/user.js

```diff
@@ -44,6 +44,8 @@ module.exports = (sequelize, DataTypes) => {
       }
     }).catch(error => databaseError(error.message));
 
+  User.findUser = email => User.findOne({ where: { email } }).catch(error => databaseError(error.message));
```

Comment:
`findUser` by what param? If someone were to look at this function's name without looking at the defined `email` param, they might think it works with the user `Id` too.